tests/vmi_multus_test.go

```diff
@@ -862,16 +862,16 @@ var _ = Describe("[Serial]SRIOV", func() {
 			cidrA := "192.168.1.1/24"
 			cidrB := "192.168.1.2/24"
 			vmi1, vmi2 := createSriovVMs(cidrA, cidrB)
-			Expect(libnet.PingFromVMConsole(vmi1, cidrToIP(cidrB))).To(Succeed())
-			Expect(libnet.PingFromVMConsole(vmi2, cidrToIP(cidrA))).To(Succeed())
+			Expect(libnet.PingFromVMConsole(vmi1, cidrToIP(cidrB), "-c 1", "-w 10")).To(Succeed())
```

Comment:
Could you increase these timeouts globally in the helper? It would allow us to forget that there is any difference at all.

Reply:
It will affect all the tests that are using them, is that ok ?

Reply:
It will affect them only if they are about to time-out. I think it is fair to keep it unified unless there is something explicitly needing more time. In this case, I'd just increase the global timeout. Or commented here on why we something else than the default. Up to you.